pkg/actions/nodegroup/get.go

```diff
@@ -151,6 +129,34 @@ func (m *Manager) Get(name string) (*manager.NodeGroupSummary, error) {
 	}, nil
 }
 
+func (m *Manager) getInstanceTypes(ng *awseks.Nodegroup) string {
+	if len(ng.InstanceTypes) > 0 {
+		return strings.Join(aws.StringValueSlice(ng.InstanceTypes), ",")
+	}
+
+	if ng.LaunchTemplate == nil {
+		logger.Info("no instance type found for nodegroup %q", *ng.NodegroupName)
+		return "-"
+	}
+
+	resp, err := m.ctl.Provider.EC2().DescribeLaunchTemplateVersions(&ec2.DescribeLaunchTemplateVersionsInput{
+		LaunchTemplateId: ng.LaunchTemplate.Id,
+	})
+
+	if err != nil {
+		return "-"
+	}
+
+	for _, template := range resp.LaunchTemplateVersions {
+		if strconv.Itoa(int(*template.VersionNumber)) == *ng.LaunchTemplate.Version {
+			return *template.LaunchTemplateData.InstanceType
```

Comment:
this would only return the first `InstanceType` when looping over `resp.LaunchTemplateVersions`, what if there are more? are we returning too early?

Reply:
Its not that its only the first return, but rather the lauch template version that matches the version is use.  Alaunch template has multiple versions. In this case we want to get the instance type from a given launch template & version. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html#manage-launch-template-versions

Reply:
Right! That makes sense ☺️